dll/win32/setupapi/devinst.c

```diff
@@ -5034,18 +5033,13 @@ ResetDevice(
     }
 
     return TRUE;
-#else
-    FIXME("Stub: ResetDevice(%p %p)\n", DeviceInfoSet, DeviceInfoData);
-    return TRUE;
-#endif
 }
 
 static BOOL
 StopDevice(
         IN HDEVINFO DeviceInfoSet,
         IN PSP_DEVINFO_DATA DeviceInfoData)
 {
-#ifndef __WINESRC__
```

Comment:
Keep the code and replace by `#ifdef __REACTOS__` instead.

Reply:
Fixed.